streamtrace.hh

```diff
@@ -229,6 +232,19 @@ struct capability_register
 static_assert(sizeof(capability_register) <= 32,
               "Capability register structure has grown far too big!");
 
+union debug_trace_register
```

Comment:
SWIG does not currently support nested unions. It is needed to make `debug_trace_entry::reg_value` accessible from python.